docs/content/environment_tests.md

```diff
@@ -61,21 +61,23 @@ The second optional argument, `test_kept_state` allows the user to disable the s
 
 The max cycles test tests that the `max_cycles` environment argument exists and the resulting environment actually runs for the correct number of cycles. If your environment does not take a `max_cycles` argument, you should not run this test. The reason this test exists is that many off-by-one errors are possible when implementing `max_cycles`. An example test usage looks like:
 
-``` python
+``` python notest
 from pettingzoo.test import max_cycles_test
 from pettingzoo.butterfly import pistonball_v6
 env = pistonball_v6.env()
+#TODO AttributeError: 'OrderEnforcingWrapper' object has no attribute 'parallel_env'
```

Comment:
Hmmm this seems like a weird bug all you’re doing is importing Pistonball. Definitely important to check if this is a real bug or what’s up with it